tests/test_document.py

```diff
@@ -372,3 +381,10 @@ def test_doc_splitter_images(self, reset_db: None) -> None:
 
         res = chunks.collect()
         assert all(isinstance(r['image'], PIL.Image.Image) for r in res)
+
+    def test_pdf_paragraph_splitting_not_supported(self, reset_db: None) -> None:
```

Comment:
Maybe just `test_doc_splitter_errors` (this feels like an awfully specific test name :) )